.github/workflows/build-and-test.yml

```diff
@@ -234,8 +234,9 @@ jobs:
           # on macOS: https://go.dev/doc/go1.15
           #- goos: darwin
           #  goarch: 386
-          - goos: aix
-            goarch: ppc64
+          # disabling this test until https://github.com/open-telemetry/opentelemetry-collector/issues/12668 is addressed
```

Comment:
Do we still need this? Looks like http://github.com/open-telemetry/opentelemetry-collector/pull/12674 fixed it